test/req/utils_test.exs

```diff
@@ -15,7 +15,7 @@ defmodule Req.UtilsTest do
         service: "s3",
         datetime: ~U[2024-01-01 09:00:00Z],
         method: :get,
-        url: "https://s3",
+        url: "https://s3/test/path:v1/invoke",
```

Comment:
nitpick: let's use more generic path? this one seemed related to some kind of api other than s3 maybe?

```suggestion
        url: "https://s3/foo/:bar",
```